app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
```

Comment:
Ok. Good feedback. I would prefer it as a service myself. Would it be ok to leave as is for now, and I make it a service over time? I want to replace the offline watch face really bad, but don't have much time right now. Also waiting for my new pebble in June. 

 
John Stevens
M - +61 418526349

> On 20 May 2015, at 07:56, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java:
> 
> > @@ -39,6 +40,10 @@
> >      @Column(name = "operation_type")
> >      public String operation_type;
> >  
> > +    private static Context mContext = Home.getContext();
> gotcha gotcha (and yeah, sorry github is also awful on phones)
> 
> If we are worried about creating too many we should either have it be a singleton or a service, either would be fine with me. Im more of a fan of services as they feel more android-ish but they seem to have more resource overhead
> 
> —
> Reply to this email directly or view it on GitHub.
>